### DISCUSSION TITLE
what's the different between mqtt.max_packet_size and zone.external.max_packet_size

### DISCUSSION THREAD

**Discussion by kiddyt00** [emqx/emqx#7625] [Q&A] [answered]
文档中显示两个参数都是   允许的 MQTT 报文最大长度
这俩参数有啥区别呢？我测试感觉好像起作用的是zone.external.max_packet_size
而mqtt.max_packet_size并没有起作用

**Comment by zmstone** [comment, marked as answer]
`mqtt` prefixed config is global, Zone configs override global.

**Reply by kiddyt00** [reply]
got it ,thanks
